src/foam/nanos/auth/email/EmailVerificationWebAgent.java

```diff
@@ -54,19 +54,19 @@ public void execute(X x) {
       }
 
       if ( "".equals(userId) || !StringUtils.isNumeric(userId) ) {
-        throw new Exception("User not found.");
+        throw new Exception("User not found");
       }
 
       if ( user.getEmailVerified() ) {
-        throw new Exception("Email already verified.");
+        throw new Exception("Email already verified");
       }
       user = (User) user.fclone();
       emailToken.processToken(x, user, token);
     } catch (Throwable t) {
       String msg = t.getMessage();
       ((Logger) x.get("logger")).error(msg);
       t.printStackTrace();
-      message = "Problem verifying your email.<br>" + msg;
+      message = "Problem verifying your email<br>" + msg;
```

Comment:
So for this line. Can we change it to "There was a problem verifying your email." It seems that msg is added onto the message. So you're gonna want to keep the fullstop here.